server/indexers/gh_issues.py

```diff
@@ -13,14 +13,25 @@ def index(gh_type, client, repo_name):
     indexed_timestamp = helpers.get_indexed_version(gh_type, repo_name, obj_type)
     # increment timestamp by one second to prevent duplicates
     indexed_timestamp = nudge_datetime(indexed_timestamp)
-    bulk_data = index_gh_issues(gh_type, client, repo_name, indexed_timestamp)
-    bulk_data += index_gh_issue_comments(gh_type, client, repo_name, indexed_timestamp)
-    helpers.update_repo_index(gh_type, repo_name, obj_type, bulk_data)
+    bulk_data = []
+    if is_updated_issues(client, repo_name, indexed_timestamp):
+        bulk_data += index_gh_issues(gh_type, client, repo_name)
+        bulk_data += index_gh_issue_comments(gh_type, client, repo_name)
+        helpers.rebuild_repo_index(gh_type, repo_name, obj_type, bulk_data)
+    else:
+        print '%s (%s): skipping %s' % (repo_name, gh_type, obj_type)
```

Comment:
will this print skipping and then print the line at 24? I don't think that's what we want